src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/IntermediateNodeWriter.cs

```diff
@@ -313,11 +313,9 @@ protected void WriteSourceRange(SourceSpan sourceRange)
 
             _writer.Write(")");
         }
-
-
         protected void WriteContent(string content)
```

Comment:
I might be reading the diff wrong, but isthis removing all whitespace above this method?